P1_BMI_Stats.R

```diff
@@ -603,7 +634,7 @@ hb_pop_estimates <- readRDS(paste0(
 
 hb_pop_estimates <- hb_pop_estimates %>% 
   filter(age == 5) %>%
-  filter(year >= 2000 & year <=2018) %>% 
+  filter(year >= 2001 & year <=2017) %>% 
```

Comment:
Create function for this as it's done 6 times in the script.

Reply:
Yes, I have this one on my list to add to the functions.